packages/front-end/pages/experiment/[eid].tsx

```diff
@@ -84,7 +82,7 @@ const ExperimentPage = (): ReactElement => {
   let canRunExperiment = !experiment.archived;
   const envs = getAffectedEnvsForExperiment({ experiment });
   if (envs.length > 0) {
-    if (!permissions.check("runExperiments", experiment.project, envs)) {
+    if (!permissionsUtil.canRunExperiment(experiment, envs)) {
       canRunExperiment = false;
     }
   }
```

Comment:
@jdorn Just wanted to double check this logic. We're only checking for the `canRunExperiment` permission if the experiment has environments that are affected.

In my testing, I had an experiment that wasn't linked to a feature that an `analyst` was able to start.

Is that expected? Or should we update this logic to remove the `if(envs.length > 0)` block?

Reply:
yes, that is expected